### PR TITLE
Fixes #499 Remove irrelevant data from Backend README.md

### LINKED ISSUE #499: Remove irrelevant data from Backend README.md
@**I'm submitting a ...**
- [ ] bug report
- [X] feature request

**Current behavior:**
Unnecessary data shown in backend readme.md file

**Expected behavior:**
Should be removed

**Steps to reproduce:**

### PR DESCRIPTION
**Pull request by Minkush-sopho**
Fixes #

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- 3rd point modified in introduction section as now user have to enter 5 entries, instead of earlier 4.
-   Removed Usage section, since Vagrant setup is available now. User Input Guide is not required either now.
-    Fixed installation links to correct location.
-    Fixed implementation links to correct location.
-    Fixed a link in maintainers section with text Badgeyay.

### CODE REVIEW

**Comment by Minkush-sopho** [comment]
@rupav please review :).

**Comment by Minkush-sopho** [comment]
Thanks @rupav

File: backend/README.md [hunk 1]
```diff
@@ -152,10 +148,10 @@ Also read [CONTRIBUTING.md](https://github.com/fossasia/badgeyay/blob/developmen
 Implementation
 --------------
 
-[generate_badges.py](/app/generate-badges.py) creates svg files from the `csv`, `png` and
+[generate_badges.py](app/generate_badges.py) creates svg files from the `csv`, `png` and
```
Comment (ParthS007) [inline backend/README.md]: Please refer to valid location i.e use the location of scripts present in back-end folder.

File: backend/README.md [hunk 2]
```diff
@@ -152,10 +148,10 @@ Also read [CONTRIBUTING.md](https://github.com/fossasia/badgeyay/blob/developmen
...
 [badges/8BadgesOnA3.svg](badges/8BadgesOnA3.svg).
 
-[merge_badges.py](/app/merge_badges.py) converts them into pdf files and merges
+[merge_badges.py](app/merge_badges.py) converts them into pdf files and merges
```
Comment (ParthS007) [inline backend/README.md]: same

File: backend/README.md [hunk 3]
```diff
@@ -181,7 +177,7 @@ To join the team:
 3. A Badgeyay admin adds you.
 
 To stay a maintainer in the team:
-1. Follow the rules of [CCCC][cccc] or [Badgeyay](.github/CONTRIBUTING.md) and do not violate them willingly or in a harmful way.
+1. Follow the rules of [CCCC][cccc] or [Badgeyay](../.github/CONTRIBUTING.md) and do not violate them willingly or in a harmful way.
```
Comment (ParthS007) [inline backend/README.md]: Please remove maintainer section, It is not needed now 👍

File: backend/README.md [hunk 4]
```diff
@@ -9,7 +9,7 @@
 The user can:
   * Choose size of badges
   * Choose background of badges and upload logo and background image
-  * Upload a CSV file or manually enter CSV data as: name, type of attendee, nick/handle, organization/project
+  * Upload a CSV file or manually enter CSV data as: name, type of attendee, nick/handle, organization/project, designation
```
Comment (ParthS007) [inline backend/README.md]: Please add designation before `nick/handle`.

**Comment by Minkush-sopho** [comment]
Okay I'll do that @ParthS007

File: backend/README.md (hunk 1 above)
Comment (Minkush-sopho) [reply backend/README.md]: I didn't get this. generate_badges.py is present in backend only in the app folder

Comment (Minkush-sopho) [reply backend/README.md]: @rupav

File: backend/README.md (hunk 3 above)
Comment (Minkush-sopho) [reply backend/README.md]: @ParthS007  Yeah sure!

File: backend/README.md (hunk 4 above)
Comment (Minkush-sopho) [reply backend/README.md]: Okay @ParthS007

File: backend/README.md (hunk 1 above)
Comment (ParthS007) [reply backend/README.md]: I mean to write `backend/app/generate_badges.py` as a path, I hope now you got it 👍

Comment (Minkush-sopho) [reply backend/README.md]: @ParthS007 But sir I am fixing the links of README.md present in the backend folder!

Comment (Minkush-sopho) [reply backend/README.md]: Doing this backend/app/generate_badges.py would not work I think.

Comment (Minkush-sopho) [reply backend/README.md]: If backend needs to be included in the path then I can change it with (../backend/app/generate_badges.py)

Comment (ParthS007) [reply backend/README.md]: ohh , my mistake then its LGTM 👍

Comment (Minkush-sopho) [reply backend/README.md]: Okay thanks for the time 👍

**Pull request closed without merging**